Gopkg.toml

```diff
@@ -91,6 +95,6 @@ required = [
   name = "github.com/knative/test-infra"
   non-go = false
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/knative/pkg"
+[[prune.project]]
```

Comment:
Why do we need to add this prune?

Reply:
We only keep the packages when they are used.

Reply:
In general, we prune go-tests, non-go and unused-packages. For knative.dev/pkg, we keep non-go, e.g. the license.

Reply:
sounds good